TransformationSystem/Client/TaskManager.py

```diff
@@ -474,11 +474,13 @@ def prepareTransformationTasks(self, transBody, taskDict, owner='', ownerGroup='
     """ Prepare tasks, given a taskDict, that is created (with some manipulation) by the DB
         jobClass is by default "DIRAC.Interfaces.API.Job.Job". An extension of it also works.
 
-    :param transBody: transformation job template
-    :param taskDict: dictionary of per task parameters
-    :param owner: owner of the transformation
-    :param ownerGroup: group of the owner of the transformation
-    :param ownerDN: DN of the owner of the transformation
+
+    :param str transBody: transformation job template
+    :param dict taskDict: dictionary of per task parameters
+    :param str owner: owner of the transformation
+    :param str ownerGroup: group of the owner of the transformation
+    :param str ownerDN: DN of the owner of the transformation
+    :param bool bulkSubmissionFlag: flag for using bulk submission or not
```

Comment:
you need an empty line

Reply:
Added all the empty lines.

Reply:
Actually you don't need an empty line between the `:param...` and `:return:` lines

Cf. http://lcd-data.web.cern.ch/lcd-data/doc/ilcdiracdoc/_modules/ILCDIRAC/Interfaces/Utilities/DDInterfaceMixin.html#DDInterfaceMixin.getKnownDetectorModels
and http://lcd-data.web.cern.ch/lcd-data/doc/ilcdiracdoc/DOC/ILCDIRAC/Interfaces/Utilities/DDInterfaceMixin.html#ILCDIRAC.Interfaces.API.NewInterface.Applications.DDSim.DDInterfaceMixin.getKnownDetectorModels

Reply:
What ?!?!?! But you drilled this down so hard in our head !!!! 
But good to know :-) it makes it more readable however :)

Reply:
Sorry for the confusion, I tried to be clear in the past that the combined `:param:/:return:` block needs to be separated by a line break.